lighttpd_rust_amalgamated/src/lighttpd_amalgamated.rs

```diff
@@ -1,8 +1,15 @@
 #![allow(dead_code, mutable_transmutes, non_camel_case_types, non_snake_case, non_upper_case_globals, unused_assignments, unused_mut)]
 #![feature(c_variadic, core_intrinsics, extern_types, label_break_value)]
 #[macro_use]
+
+extern crate libc;
+extern crate core;
+extern crate lighttpd_rust_amalgamated;
```

Comment:
I was wondering where this crate comes from in the first place.  I didn't realize there's a separate `lighttpd_rust_amalgamated/lib.rs`.  Could we simply move everything from `lib.rs` and `lighttpd_amalgamated.rs` into `main.rs` along with a normal `Cargo.toml`?  Since `c2rust-analyze` runs on a single file, the full crate and single file behavior will become the same if there's only a single file.

Reply:
And `lib.rs` already has `extern crate libc;`, added here, and `#[macro_use] extern crate c2rust_bitfields;`, which is equivalent to the `extern crate` and `use c2rust_bitfields::*;`.

Reply:
let me know if ae640e3 doesn't cover everything